cli/lsp/diagnostics.rs

```diff
@@ -367,6 +367,15 @@ async fn generate_ts_diagnostics(
     let ts_diagnostics_map: TsDiagnosticsMap =
       ts_server.request(snapshot.clone(), req).await?;
     for (specifier_str, ts_diagnostics) in ts_diagnostics_map {
+      // Code 2688 represents "Cannot found type definitions file" diagnostic. We want
+      // to skip it. See https://github.com/denoland/deno/issues/10575
+      let mut ts_diagnostics = ts_diagnostics;
+      let type_definition_not_found =
+        ts_diagnostics.iter().position(|d| d.code == 2688);
+      if let Some(index) = type_definition_not_found {
+        ts_diagnostics.remove(index);
+      }
+
```

Comment:
@kitsonk any suggestion on where I should add a test for it (or can we skip it)? I feel like an integration test would add too much weight for this small change?